arrow/src/main/java/org/apache/iceberg/arrow/vectorized/parquet/VectorizedPageIterator.java

```diff
@@ -512,6 +512,9 @@ protected void initDataReader(Encoding dataEncoding, ByteBufferInputStream in, i
         throw new ParquetDecodingException("could not read page in col " + desc, e);
       }
     } else {
+      if (dataEncoding != Encoding.PLAIN) {
+        throw new UnsupportedOperationException("Unsupported encoding: " + dataEncoding);
```

Comment:
I would like to emphasize that a user can use non-vectorized reads to handle this file so maybe something like

"Cannot perform a vectorized read of ParquetV2 File with encoding %s, disable vectorized reading with $param to read this table/file"

Reply:
Actually since we may get users who read some columns successfully but fail on others we probably should be specific about which column failed in the error message as well. Just so someone doesn't say
"When i do this projection it works, but when I do this projection it doesn't"

Reply:
I like the idea of specifying the column name and being more descriptive about why we are failing. However, there are different ways to disable vectorization using table properties, spark session properties etc. For now, I am going with something like this:
```
if (dataEncoding != Encoding.PLAIN) {
        throw new UnsupportedOperationException("Vectorized reads are not supported for column " + desc + " with " +
            "encoding " + dataEncoding + ". Disable vectorized reads  to read this table/file");
      }
```

Reply:
Sounds good to me, I do know most of the time we have errors styled as "Cannot X " but I think your content suggestion for the error message is solid. I would just add that so it fits with the other messages.